### DISCUSSION TITLE
Active Learning on Image Tasks

### DISCUSSION THREAD

**Discussion by HridayM25** [keras-team/keras-cv#1442] [Ideas]
Hello, is there any provision of active learning on image tasks offered by KerasCV?
I feel that some work could be done to include this, as this is a significant concept to reduce labour?
There was some discussion previously opened on the same topic, but I couldn't see any development happening there.
Thanks

**Comment by jbischof** [comment]
@HridayM25 do you have any examples/papers about what you're trying to do? I know about active learning generally but not in a way that our library would benefit from.
